### PR TITLE
Stub SHIELD study consent page

### PR DESCRIPTION
**Pull request by brittanystoroz**
I'm a jerk and accidentally merged https://github.com/mozilla/normandy-addon/pull/29. Sorry! 

I reverted the commits and am opening this PR, which includes feedback from the original review. (Use yo-yo, remove all the IDs in the HTML, remove licensing information, move eslint globals into the data dir).

### CODE REVIEW

**Comment by mythmon** [comment]
Ah, ok! I was wondering what you did with #29. I'll look over this.

**Comment by mythmon** [comment]
We might be able to clean up the git history a bit here?

File: data/consent-page.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+const buildStudyTitle = studyName => {
+  const titleText = studyName || 'Participate in a new Firefox Study';
+  return yo`<h1>${titleText}</h1>`;
+};
+
+// Build the opt-in button
+const buildParticipateButton = (buttonText, buttonClasses) => {
+  const text = buttonText || `Try ${self.options.addonName}`;
+  return yo`
+  <button id="install-addon" class=${buttonClasses || 'primary'}>
```
Comment (mythmon) [inline data/consent-page.js]: Nit: I think we should write this as

```
return yo`
  <button id="install-addon" class=${buttonClasses || 'primary'}>
    ${text}
  </button>
`;
```

That is, a line ending with a backtick should indent the next line, and should be dedented at the end. I think that lines up better with the rest of our styling.

Comment (mythmon) [inline data/consent-page.js]: Separately, I think we need to have `class="${buttonClasses || 'primary}"`, ie: with quotes around the classname. Or does yoyo deal with that for us?

File: data/consent-page.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+// Create studyTitle, button, and authorsList
+const studyTitle = buildStudyTitle(self.options.studyName);
+const participateButton = buildParticipateButton(self.options.buttonText);
+const authorsList = buildAuthorsList(self.options.authors);
+
+
+// Append header & button
+const headerElement = document.getElementById('study-header');
+      headerElement.appendChild(studyTitle);
+      headerElement.appendChild(participateButton);
```
Comment (mythmon) [inline data/consent-page.js]: 🙁 indentation that doesn't represent structure makes things confusing. I am surprised this passed linting. (are lints being run on this repo?)

File: package.json [hunk 3]
```diff
@@ -2,14 +2,15 @@
...
-  "version": "0.1.0",
+  "version": "0.1.3",
   "description": "Client to download and run recipes for SHIELD, Heartbeat, etc.",
   "main": "index.js",
   "author": "Mike Cooper <mcooper@mozilla.com>",
   "repository": "https://github.com/mozilla/normandy-addon",
   "scripts": {
     "test": "jpm test",
-    "build": "jpm xpi"
+    "webpack": "node_modules/.bin/webpack data/consent-page.js data/dist/consent-page.js",
```
Comment (mythmon) [inline package.json]: npm scripts set up the path nicely, so you don't have to do `node_modules/.bin/`, you can just say `webpack`.

**Comment by brittanystoroz** [comment]
Fixed feedback and cleaned up the git commit history.

File: data/consent-page.js (hunk 2 above)
Comment (brittanystoroz) [inline data/consent-page.js]: I ran lints, I think they showed up as warnings rather than errors and I had left them be.  Fixed.

Comment (mythmon) [inline data/consent-page.js]: Ah. I forget that warnings and errors exist separately.

File: data/.eslintrc.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+extends: 'normandy'
```
Comment (mythmon) [inline data/.eslintrc.yaml]: Sorry I wasn't clearer. This file will automatically build on top of the one in the parent directory. So you'll only have to include the changes for this directory. I imagine the whole file could be something like

```
globals:
  document: true
  self: true
```

**Pull request merged**